pkg/services/store/types.go

```diff
@@ -95,3 +95,40 @@ type RootStorageMeta struct {
 
 	Config RootStorageConfig `json:"config"`
 }
+
+type StorageListFrame struct {
+	*data.Frame
+}
+
+type listFrameField string
```

Comment:
AFAICT, we're casting every one of these back to string on every use, what does this type give us?